data/is-cfg.json

```diff
@@ -76,8 +76,8 @@
 		"topic": "LoraAPRS/Data"
 	},
 	"syslog": {
-		"active": true,
-		"server": "syslog.lora-aprs.info",
```

Comment:
please re-add the syslog server for people which do not have one. as this feature will be removed soon (good idea, but in the end it was not helping a lot) it can be de-activated per default.

Reply:
Ok, thank you for your feedback. Is there a way to have access to this syslog? I do not think I saw anything about it online.